test/testrunner.py

```diff
@@ -68,7 +68,7 @@ def setup_eks_cluster(framework_name):
     long_name = framework_name
     short_name = frameworks[long_name]
     codebuild_version = os.getenv('CODEBUILD_RESOLVED_SOURCE_VERSION')[0:7]
-    num_nodes = 1 if is_pr_context() else 3 if long_name != "pytorch" else 4
```

Comment:
Are we now running multinode test for PR  as resource utilization will increase a lot.

Reply:
Oh Sorry is it for testing purpose right now ?

Reply:
Yup. Thanks. It has only been enabled for the sake of this PR. I will revert it after verifying that this Sequencing works for all images and frameworks on PRs, and getting successful runs for sequenced Multi-Node EKS tests.